src/lib/layouts/Canvas.svelte

```diff
@@ -44,5 +53,8 @@
 	style:bottom={$padding.bottom + 'px'}
 	style:left={$padding.left + 'px'}
 	style="width:100%;height:100%;position:absolute;"
-></canvas>
+	aria-label={label}
+	aria-labelledby={labelledBy}
+>{fallbackText}</canvas>
+<slot name="title"></slot>
```

Comment:
This seems weird to have it as a slot here. Should there be a title on the canvas component?